src/main/resources/lib/credentials/select/select.js

```diff
@@ -260,6 +260,17 @@ Behaviour.specify("INPUT.credentials-select", 'credentials-select', -100, functi
   }).bind($(x));
   x.onchange();
 });
+Behaviour.specify("DIV.include-user-credentials", 'include-user-credentials', 0, function (e) {
+    e.querySelector("input[name='includeUser']").onclick = function (evt) {
+        var caution = e.querySelector('span.user-credentials-caution');
+        caution.hidden = !this.checked;
```

Comment:
FTR the hidden attribute is valid under IE11, Chrome (since 6), Firefox (since 4).